docs/src/components/navigation/keystatic-logo.tsx

```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link';
+
+export function KeystaticLogo() {
+  return (
+    <Link href="/">
```

Comment:
We should probably put an `aria-label` on here. What do you think about: `Home of Keystatic`?

Reply:
And maybe move the link out to the parent?